packages/optimism-provider/test/data/ERC20.json

```diff
@@ -0,0 +1,718 @@
+{
```

Comment:
Is this something we should expose in the contracts repository? Happy to have it here though, just curious